src/set.rs

```diff
@@ -288,18 +288,22 @@ where
     ///
     /// let a = HashSet::from_iter(&[1, 2, 3]);
     /// let b = HashSet::new();
-    /// let guard = a.guard();
     ///
-    /// assert_eq!(a.is_disjoint(&b, &guard), true);
-    /// b.insert(4, &guard);
-    /// assert_eq!(a.is_disjoint(&b, &guard), true);
-    /// b.insert(1, &guard);
-    /// assert_eq!(a.is_disjoint(&b, &guard), false);
-    ///
-    /// ```
-    pub fn is_disjoint(&self, other: &HashSet<T, S>, guard: &Guard) -> bool {
-        for value in self.iter(guard) {
-            if other.contains(&value, guard) {
+    /// assert_eq!(a.pin().is_disjoint(&b.pin()), true);
```

Comment:
These can just be `assert!` instead of `assert_eq!` :)

Reply:
Sure. I was copying the examples from [`std::collections::HashSet::is_disjoint`](https://doc.rust-lang.org/std/collections/struct.HashSet.html#method.is_disjoint), which use `assert_eq` instead of `assert`.